source/install/troubleshooting.rst

```diff
@@ -629,3 +631,300 @@ To help us narrow down whether it’s a server configuration issue, device speci
+
+.. code-block:: none
+   
+   tar -czf /tmp/mattermost.log.tgz
+
+The compressed logs will be located on the server at ``/tmp/mattermost.log.tgz``.
+
+If the compressed file is still too big, use these commands to split the compressed file into two or more 20MB files:
+
+.. code-block:: none
+   
+   mkdir -p /tmp/mattermost-logs
+   cd /tmp/mattermost-logs
+   tar czf - /opt/mattermost/logs/mattermost.log | split -b 20m - mattermost.log.tgz.
+
+The compressed files will be located on the server at ``/tmp/mattermost-logs`` and be named ``mattermost.log.tgz.aa``, ``mattermost.log.tgz.ab``, and so on. Use a file transfer client that supports SSH/SFTP, such as Cyberduck, to copy these files from the server.
+
+If you are experiencing issues with Elasticsearch, SAML, LDAP, or the database, you can enable trace logging in ``config.json`` by setting ``Trace`` to ``true`` under their respective settings. Combining this with ``DEBUG`` level file log output will result in huge log files, so only leave trace logging on long enough to replicate the behavior. The resulting logs will also contain a lot more sensitive data, including user data, so be sure to sanitize it completely before sharing it with us.
+
+**System logs**
+
+The location of log files for other systems varies, but a good way to get the logs for all processes on the Mattermost server is to use ``journalctl`` like this:
+
+.. code-block:: none
+  
+   sudo journalctl --since "2020-08-23 17:15:00" > mattermost_journalctl.log
+
+Replace 2020-08-23 17:15:00` with the date and time (relative to the server) when the error occurred. You can use ``--until`` with the same timestamp format to get the logs between two times:
+
+.. code-block:: none
+  
+   sudo journalctl --since "2020-08-23 17:15:00" --until "2020-08-23 16:30:00" > mattermost_journalctl.log
+
+Users affected
+~~~~~~~~~~~~~~
+
+Why we need it
+^^^^^^^^^^^^^^
+
+Mattermost servers are chaotic places. Thousands of posts, websocket actions, and webhook calls happen every second while users can be in dozens of channels across multiple teams. Knowing which users are affected by a problem can help us sift through all this information to find the root cause.
+
+What information to include
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This should be a detailed explanation of anything the end users who are reporting the unexpected behavior have in common. This includes (but is not limited to):
+
+- Team and Channel memberships, including Direct and Group Messages
+- Authentication methods
+- Client operating system and app versions
+- How users connect to the Mattermost server
+- Any other things these users have in common such as when they joined, whether their login information recently changed, or if they are being synchronized via LDAP
+
+Note for Agents: This information is also required:
+
+- Customer name
+- Customer contacts
+- Customer license, e.g. E20/PS
+- Customer tier
+
+Environment
+~~~~~~~~~~~
+
+What information to include
```

Comment:
This heading is duplicated above.

Reply:
I think that's intentional but I've moved it to beneath the opening sentence for readability. LMK if I completely misunderstood!